contributing.md

```diff
@@ -545,14 +545,14 @@ You can find more details [here](https://github.com/colcon/colcon-sanitizer-repo
 
 #### Compiling the code and running the tests
 
-Create a Ignition workspace for ASAN/TSAN related tasks. Refer to the [offical documentation to get
-all the Ignition Robotics sources](https://gazebosim.org/docs/latest/install_ubuntu_src#getting-the-sources).
+Create a Gazebo workspace for ASAN/TSAN related tasks. Refer to the [offical documentation to get
+all the Gazebo Robotics sources](https://gazebosim.org/docs/latest/install_ubuntu_src#getting-the-sources).
```

Comment:
Probably?

```suggestion
all the Gazebo sources](https://gazebosim.org/docs/latest/install_ubuntu_src#getting-the-sources).
```

Reply:
We are not using `Robotics` as a modifier to `Gazebo`, right @chapulina?

Reply:
Yeah nice catch @methylDragon , we've been dropping the Robotics everywhere.

Reply:
Thanks, removed.